returnn/tf/layers/signal_processing.py

```diff
@@ -387,33 +437,34 @@ def __init__(self, frame_shift, frame_sizes, fft_sizes, window="hanning", use_rf
     :param list(int) fft_sizes: fft size in samples
     :param str window: id of the windowing function used. Possible options are:
       - hanning
-    :param bool use_rfft: if set to true a real input signal is expected and only
-      the significant half of the FFT bins are returned
+    :param bool use_rfft: if set to true a real input signal is expected and only the significant half of the FFT bins
+    are returned
     :param int nr_of_channels: number of input channels
-    :param bool pad_last_frame: padding of last frame with zeros or discarding of
-      last frame
+    :param bool pad_last_frame: padding of last frame with zeros or discarding of last frame
     """
+
     def _compute_size_placeholder():
       size_placeholder_dict = {}
       nr_of_full_frames = (self.input_data.size_placeholder[0] - self._reference_frame_size) // self._frame_shift + 1
       nf_of_paded_frames = 0
-      if self._pad_last_frame and ((self.input_data.size_placeholder[0] - self._reference_frame_size) - (nr_of_full_frames - 1) * self._frame_shift > 0):
+      if self._pad_last_frame and ((self.input_data.size_placeholder[0] - self._reference_frame_size) -
+                                   (nr_of_full_frames - 1) * self._frame_shift > 0):
```

Comment:
You should never indent like that (such wide to the right). Introduce further brackets, and then you can easily indent right after `if (`.